src/runtime/pkg/resourcecontrol/utils.go

```diff
@@ -80,3 +82,33 @@ func IsSystemdCgroup(cgroupPath string) bool {
 
 	return false
 }
+
+func IsCgroupV1() (bool, error) {
+	if cgroups.Mode() == cgroups.Legacy || cgroups.Mode() == cgroups.Hybrid {
+		return true, nil
+	} else if cgroups.Mode() == cgroups.Unified {
+		return false, nil
+	} else {
+		return false, ErrCgroupMode
+	}
+}
+
+// SandboxAndOverheadPath gets the cgroup path in thread mode in cgroup v2.
+// The sandbox and overhead cgroup in threaded mode are placed under the same cgroup in domain threaded mode.
+// In this way, vCPU threads and VMM processes can be separated into two cgroups.
+// For details, please refer to https://github.com/kata-containers/kata-containers/issues/4886.
+func SandboxAndOverheadPath(cgroupPath string, overheadPath string, sandboxCgroupOnly bool) (string, string, error) {
```

Comment:
Input parameter `overheadPath` is not used, so it can be deleted.

Reply:
Tough `cgroupPath` is used, but changing the input parameter and returning it is not a good design pattern.

Reply:
I modified how the parameters are used and added comments for two cases.